### PR TITLE
#888 Add configuration snippets for Gradle to reference guide and to …

### PR DESCRIPTION
**Pull request by agudian**
…the readme.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -72,6 +72,22 @@ For Maven based projects add the following to your POM file in order to use MapS
...
+```groovy
+plugins {
+    ...
+    id 'net.ltgt.apt' version '0.8'
+}
+dependencies {
+    ...
+    compile 'org.mapstruct:mapstruct:1.0.0.Final' // OR use this with Java 8 and beyond: org.mapstruct:mapstruct-jdk8:...
+
+    apt 'org.mapstruct:mapstruct-processor:1.0.0.Final'
```
Comment (sjaakd) [inline readme.md]: Is there any way we can get away with not specifying version.. Or stating somehow this is an example?

**Comment by sjaakd** [comment]
@agudian .. Only one comment..Looks good otherwise.

File: readme.md (hunk 1 above)
Comment (agudian) [inline readme.md]: We'll need to run a search/replace on the readme after a Final release anyway (it's up higher up in the docs already...). I don't know of a way to extract that to a single source.

**Comment by agudian** [comment]
Thanks for taking a look! Your PR can go in first, I'll rebase then... :)

**Comment by sjaakd** [comment]
> Your PR can go in first,

Done..

**Pull request merged**